src-tools/CrosshairComponent.cpp

```diff
@@ -250,48 +269,90 @@ void dm::CrosshairComponent::mouseDown(const MouseEvent & event)
 
 void dm::CrosshairComponent::mouseUp(const MouseEvent & event)
 {
-	mouse_previous_loc	= mouse_current_loc;
-	mouse_current_loc	= event.getPosition() + juce::Point<int>(mouse_drag_offset.x, mouse_drag_offset.y);
+	if (panning_active)
```

Comment:
Note the CrosshairComponent is the base class for the right-hand-side scrollfield, as well as the main component canvas.  It definitely should not have any of this functionality, since the scrollfield shouldn't be scrollable.  These changes should instead be in Canvas or Content.  (Probably Canvas.)